PowerEditor/src/TinyXml/tinyxml.cpp

```diff
@@ -551,20 +551,37 @@ void TiXmlElement::SetAttribute( const TCHAR * name, const TCHAR * _value )
 	}
 }
 
-void TiXmlElement::Print( FILE* cfile, int depth ) const
+const char one_ws[] = " ";
+const char four_ws[] = "    ";
+const char eol[] = "\r\n";
+
+const generic_string tagOpenSymb = TEXT("<");
+const generic_string tagOpenSlash = TEXT("</");
+const generic_string tagCloseSymb = TEXT(">");
+
+const char tagCloseChar[] = ">";
+const char tagCloseSymbFinal[] = " />";
+
+void TiXmlElement::Print( CFile& cfile, int depth ) const
 {
 	int i;
 	for ( i=0; i<depth; i++ )
 	{
-		generic_fprintf( cfile, TEXT("    ") );
+		cfile.Write(four_ws, 4);
+		//generic_fprintf( cfile, TEXT("    ") );
```

Comment:
It's better to remove code rather than comment it (here and below). You can always find it in Git history.

Reply:
It's for the future debug need, especially for such word to word modification.
So I prefer to keep it.